app/build.gradle

```diff
@@ -20,6 +20,7 @@ android {
 }
 dependencies {
     compile files('libs/android-json-rpc-0.3.4.jar')
+    compile 'org.jsoup:jsoup:1.11.2'
```

Comment:
Yes really all screen-scraping sites should use this...

Reply:
Stay tuned. I have another PR coming that adds an abstract HTML adapter.
The PR includes a port of TPB adapter plus another private tracker I use.